docs/submission/submitting-molecular-data.md

```diff
@@ -137,6 +220,8 @@ If your payload is not formatted correctly, you will receive an error message de
 }
 ```
 
+### 3. Generate a manifest file
+
 Use the returned `analysis_id` to generate a manifest for file upload using the song-client `manifest` command. This manifest will be used with the score-client in the next step.

```

Comment:
Use the returned `analysis_id` from step 2 to generate...

(just consistent with what you did in next step)